brave/brave-core
# Fix 5869: Google OAuth should work by default

Fix https://github.com/brave/brave-browser/issues/5869

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

1. Navigate to expensify.com 
2. Verify that Cross-Site cookie blocking is enabled
3. Click login using Google
4. Login should succeed
5. Navigate to `brave://settings/socialBlocking` and disable Google login
6. Navigate to `brave://settings/clearBrowserData` and clear all data
7. Try logging in again, login should fail

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
